### PR TITLE
Iot Hub .Net SDK

### PR DESCRIPTION
**Pull request by rkmanda**
Iot Hub .Net SDK

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@rkmanda__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci retest this please

File: src/ResourceManagement/IotHub/Microsoft.Azure.Management.IotHub/generate.cmd [hunk 1]
```diff
@@ -0,0 +1,19 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+setlocal
+set autoRestVersion=0.14.0-Nightly20160125
```
Comment (hovsepm) [inline src/ResourceManagement/IotHub/Microsoft.Azure.Management.IotHub/generate.cmd]: >0.14.0-Nightly20160125 [](start = 20, length = 22)

Please use latest generator. IT should be 0.16.x

File: src/ResourceManagement/IotHub/Microsoft.Azure.Management.IotHub/project.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+  "version": "1.0.0",
+  "title": "Microsoft Azure Key Vault Management",
+  "description": "Provides management capabilities for Microsoft Azure IotHub.",
```
Comment (hovsepm) [inline src/ResourceManagement/IotHub/Microsoft.Azure.Management.IotHub/project.json]: Title should also state about your your library and not Key Vault.

**Comment by hovsepm** [comment]
:shipit:

**Pull request merged**